install/bootstrap_livecd_remote_iso.md

```diff
@@ -417,21 +417,21 @@ Some files are needed for generating the configuration payload. See these topics
 
    1. Check for the configuration files. The needed files should be in the current directory.
 
-      > **`NOTE`** for those more familiar with a CSM Install, this step may be skipped entirely by simple invoking pit-intas detailed in the [#first time](#bring---up-the-pit-services-and-validate-pit-health) section.
+      > **`Note:`** for those more familiar with a CSM Install, this step may be skipped entirely by simple invoking pit-intas detailed in the [#first time](#bring---up-the-pit-services-and-validate-pit-health) section.
 
       ```bash
       pit:/var/www/ephemeral/prep/# ls -1
       ```
 
-   Expected output looks similar to the following:
+       1. Expected output looks similar to the following:
```

Comment:
This shouldn't be a separate sub-step as it is just an example of output and the user is not doing an action. It should just be indented under the step above.